frontend/src/metabase/writeback/components/ActionCreator/FormCreator/FormCreator.tsx

```diff
@@ -90,7 +90,7 @@ function FormItem({
 
   const hasOptions =
     fieldSettings.inputType === "dropdown" ||
-    fieldSettings.inputType === "inline-select";
+    fieldSettings.inputType === "radio";
```

Comment:
nit: I have a feeling it might be handy to store utils like `hasOptions`, `hasPlaceholder`, and I'm sure something else in their own file. No immediate need now tho IMO